docs/source/nextflow-workflow.rst

```diff
@@ -135,6 +135,143 @@ Example ``sample_sheet.csv``
+
+If you are using a Slurm scheduler, you will need to create a configuration file. If you do not have a scheduler, skip ahead to "Running Autometa".
+
+First you will need to know the name of your slurm partition. Run :code:`sinfo` to find this. In the example below, the partition name is "queue".
+
+.. image:: ../img/slurm_partitions.png
+
+Next, generate a new file called slurm_nextflow.config via nano:
+
+.. code-block:: bash
+
+    nano slurm_nextflow.config
+
+Then copy the following code block into that new file:
+
+.. code-block:: bash
+
+    profiles {
+            slurm {
+                process.executor       = "slurm"
+                process.queue = "queue" // <<-- change this to whatever your partition is called
+                // queue is the slurm partition to use in our case
+                docker.enabled         = true
+                docker.userEmulation   = true
+                singularity.enabled    = false
+                podman.enabled         = false
+                shifter.enabled        = false
+                charliecloud.enabled   = false
+                executor {
+                    queueSize = 8
+                }
+            }
+        }
+
+Keep this file somewhere central to you. For the sake of this example I will be keeping it in a folder called "Useful scripts" in my home directory e.g. :code:`/home/me/Useful_scripts/slurm_nextflow.config`
+
+Save your new file with Ctrl+O and then exit nano with Ctrl+O. Installation and set up is now complete.
+
+Running Autometa
+*******************
+For a comprehensive list of features and options and how to use them please `click here <https://autometa.readthedocs.io/en/latest/nextflow-workflow.html?highlight=slurm#running-the-pipeline>`_
+
+Autometa can bin one or several metagenomic datasets in one run. Regardless of the number of metagenomes you want to process, you will need to provide a sample sheet which specifies the name of your sample, the full path to where that data is found. If the metagenome was assembled via SPAdes, Autometa can extract coverage and contig length information from the sequence headers. If you used a different assembler you will need to provide either raw reads for coverage calculation or a table of contig/scaffold coverage. Full details for data preparation can be `found here <https://autometa.readthedocs.io/en/latest/nextflow-workflow.html#sample-sheet-preparation>`_
+
+First ensure that your Autometa conda environment is activated. You can activate your environment by running:
+
+.. code-block:: bash
+    conda activate autometa-nf
+
+Navigate to where your metagenome files are and run the following code to launch Autometa:
+
+.. code-block:: bash
+
+    nf-core launch KwanLab/Autometa
+    
+You will now use the arrow keys to move up and down between your options and hit your "Enter" or "Return" key to make your choice.
+
+Menu 1:
```

Comment:
I think this would be a fab idea if each menu had a title of sorts. I could add that in I suppose e.g. Menu1 : Version choice, Menu2: Interface choice etc etc? I'll do that but can change it back if you don't like it :) Not sure about adding each one to the index though? I don't think its entirely necessary but as the boss you could persuade me otherwise if you feel strongly on this point?

Reply:
Right, so adding a title of sorts to each menu with a properly formatted header will generate an index for the section. So I guess I meant this adding these headers will implicitly add an index for the section.

For example, by adding this, the navigation along the left would have a `+` button next to `running Autometa` under `Quick Start`

Similar to some other sections with expandable sub-sections (like what is shown in the screenshot for `Basic`, `Advanced` and `Data Preparation`):

![Screen Shot 2022-02-15 at 12 44 00](https://user-images.githubusercontent.com/25933122/154118851-cc59cd47-cb6a-43ac-977f-35ff8224c57f.png)